recipes-mender/mender/mender_0.1.bb

```diff
@@ -30,18 +30,19 @@ do_compile() {
   PATH="${B}/bin:$PATH"
   export PATH
 
-  cd "${S}"
-  godep go build -o "${B}/mender"
+  cd ${S}
+  godep go build -o ${B}/mender
```

Comment:
On Fri, May 20, 2016 at 1:25 PM, Kristian Amlie <notifications@github.com>
wrote:

> In recipes-mender/mender/mender_0.1.bb
> <https://github.com/mendersoftware/meta-mender/pull/43#discussion_r64026629>
> :
>
> > @@ -30,18 +30,19 @@ do_compile() {
> >    PATH="${B}/bin:$PATH"
> >    export PATH
> >
> > -  cd "${S}"
> > -  godep go build -o "${B}/mender"
> > +  cd ${S}
> > +  godep go build -o ${B}/mender
>
> Why remove quotation? It cannot harm, only help.
>
Consistency. Nobody uses quotations here because paths are already sane.
Anyone seeing quotations will automatically assume that there is something
funky special happening here (while there's not eally)

-- 
Maciej Borzecki